pyproject.toml

```diff
@@ -10,25 +10,24 @@ packages = [
 ]
```

Comment:
These duplicate packages should be removed -- additionally, the entire package should be included, not just a single file.

Reply:
@caldwella2